extensions/amp-story/0.1/media-pool.js

```diff
@@ -241,6 +301,55 @@ export class MediaPool {
     });
   }
 
+  /**
+   * @param {!HTMLMediaElement} mediaEl
+   * @private
+   */
+  bubbleEvents_(mediaEl) {
+    // TODO(alanorozco): Keep track of unlisteners for disposal.
+    const onNonBubblingEvent = this.onNonBubblingEvent_.bind(this, mediaEl);
+    NON_BUBBLING_EVENTS.forEach(type =>
+      listen(mediaEl, type, onNonBubblingEvent.bind(this, type)));
+  }
+
+  /**
+   * @param {!HTMLMediaElement} mediaEl
+   * @param {string} type
+   * @param {!Event} event
+   * @private
+   */
+  onNonBubblingEvent_(mediaEl, type, event) {
+    if (event.bubbles) {
+      // In case of platform differences.
+      return;
+    }
+    const {parentNode} = mediaEl;
+    if (!parentNode) {
+      // Detached, can't bubble.
+      return;
+    }
+    if (type == 'volumechange') {
```

Comment:
I'm okay with this, but why are we adding `muted`?  And is it a problem that if there are platform differences causing `volumechange` to bubble by default, it won't get `muted` added?

Reply:
To satisfy the `VideoInterface`, we need to trigger mute events. In the normal amp-video implementation this happens on volume change.

I figured it would be easier to check using `getMediaInfo` rather than special-casing this event.